runelite-client/src/test/java/net/runelite/client/plugins/slayer/SlayerPluginTest.java

```diff
@@ -705,6 +707,60 @@ public void testCombatBraceletUpdate()
 		assertEquals(30, slayerPlugin.getAmount());
 	}
 
+	@Test
+	public void testTaskLookupFromNPCChat() throws IOException
```

Comment:
Is this meant to throw IOException?

Reply:
This is to be consistent with https://github.com/runelite/runelite/pull/11128/files#diff-e1297a0ae6bdf66b5be1b855ea54e07bR765 since they are testing the same code. 

I can reorder the tests to make this a bit more clear.